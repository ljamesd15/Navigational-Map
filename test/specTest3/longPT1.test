# This is in attempt to find paths relatively long pt1

# Load graph
LoadGraph marvel marvel.tsv

# Find the path
FindPath marvel SPARX/AURELLE SPECTRA/SELENA_SLATE