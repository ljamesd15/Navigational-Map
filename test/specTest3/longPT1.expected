# This is in attempt to find paths relatively long pt1

# Load graph
loaded graph marvel

# Find the path
path from SPARX/AURELLE to SPECTRA/SELENA SLATE:
SPARX/AURELLE to SERSI/SYLVIA with weight 1.000
SERSI/SYLVIA to CAPTAIN AMERICA with weight 0.012
CAPTAIN AMERICA to SPIDER-MAN/PETER PAR with weight 0.007
SPIDER-MAN/PETER PAR to SLEEPWALKER with weight 0.167
SLEEPWALKER to SPECTRA/SELENA SLATE with weight 0.500
total cost: 1.686